Nodejs/Product/Nodejs/Commands/ImportWizardCommand.cs

```diff
@@ -81,7 +81,15 @@ class ImportWizardCommand : Command {
                         }
                         if (File.Exists(path)) {
                             object outRef = null, pathRef = "\"" + path + "\"";
-                            NodejsPackage.Instance.DTE.Commands.Raise(VSConstants.GUID_VSStandardCommandSet97.ToString("B"), (int)VSConstants.VSStd97CmdID.OpenProject, ref pathRef, ref outRef);
+                            var addExistingProjectCmd = NodejsPackage.Instance.DTE.Commands.Item("File.AddExistingProject");
+                            if (addExistingProjectCmd.IsAvailable)

```

Comment:
might need a null check here to handle cases that the command doesn't exist altogether.